std/json.d

```diff
@@ -651,13 +695,13 @@ Params:
     maxDepth = maximum depth of nesting allowed, -1 disables depth checking
     options = enable decoding string representations of NaN/Inf as float values
 */
-JSONValue parseJSON(T)(T json, int maxDepth = -1, JSONOptions options = JSONOptions.none)
+JSONValue parseJSON(T)(T json, int maxDepth = -1, JSONOptions options = JSONOptions.none) @safe
```

Comment:
I already had that unittest. I've added a similar one for a @system range (and fixed the problems it revealed).